markdown-mode.el

```diff
@@ -9613,9 +9613,11 @@ rows and columns and the column alignment."
   (add-hook 'kill-buffer-hook #'markdown-live-preview-remove-on-kill t t))
 
 ;;;###autoload
-(add-to-list 'auto-mode-alist '("\\.markdown\\'" . markdown-mode))
-;;;###autoload
-(add-to-list 'auto-mode-alist '("\\.md\\'" . markdown-mode))
+(setq auto-mode-alist
```

Comment:
this looks strange - why do we reset the entire auto-mode-alist?

i prefered the previous way of doing multiple add-to-list (as in the readme) or using a regex

Reply:
Oh yes, very important *not* to set `auto-mode-alist` here, you'll break literally everyone's Emacses.

Reply:
well in this case we `setq` with the return value of `append` so it might just work. but yes, bad practice, i'd say.

Reply:
I see. Although this will add duplicates to `auto-mode-alist` whenever the file is evaluated. Defo better to use `add-to-list` (because it checks for duplicates first).